stripe.js

```diff
@@ -0,0 +1,21 @@
+var Boom = require('boom');
+var process_charge = require('./lib/stripe').process_charge;
+
+/*
+Configure stripe to only hit this endpoint with the following events:
+- charge.succeeded
+- charge.captured
+- charge.failed
+- charge.refunded
+- charge.updated
+*/
+
+module.exports = function(request, reply) {
+  var event = request.payload;
```

Comment:
Could you add a check here that the event type you're receiving is the type you expect? And just reply() immediately if it isn't.